client/src/modules/general-ledger/general-ledger-accounts.ctrl.js

```diff
@@ -25,30 +25,105 @@ function GeneralLedgerAccountsController(GeneralLedger, Session, Notify,
       displayName      : 'TABLE.COLUMNS.ACCOUNT',
       enableFiltering  : true,
       cellTemplate     : '/modules/general-ledger/templates/account_number.cell.html',
-      headerCellFilter : 'translate',
-      width            : '10%' },
+      headerCellFilter : 'translate' },
 
     { field            : 'label',
       displayName      : 'TABLE.COLUMNS.LABEL',
       cellTemplate     : '/modules/general-ledger/templates/account_label.cell.html',
       enableFiltering  : true,
       headerCellFilter : 'translate' },
 
-    { field            : 'debtor_sold',
-      displayName      : 'TABLE.COLUMNS.DEBTOR_SOLD',
+    { field            : 'balance',
+      displayName      : 'TABLE.COLUMNS.BALANCE',
+      enableFiltering  : false,
+      headerCellFilter : 'translate',
+      headerCellClass  : 'text-center',
+      cellTemplate     : '/modules/general-ledger/templates/balance.cell.html' },
+
+    { field            : 'balance1',
+      displayName      : 'TABLE.COLUMNS.DATE_MONTH.JANUARY',
+      enableFiltering  : false,
+      headerCellFilter : 'translate',
+      headerCellClass  : 'text-center',
+      cellTemplate     : '/modules/general-ledger/templates/balance1.cell.html' },
```

Comment:
Hmm... There must be an easier way to do this.

Normally, you should use an HTML template, but all these HTML templates are identical.

Try this - it will save a lot of code/HTTP requests.
```js
function getCellTemplate(key) {
  return '<div class="ui-grid-cell-contents text-right">' +
    '<div ng-show="row.entity.' + key +'" >' +
      '{{ row.entity.' + key +' | currency: grid.appScope.enterprise.currency_id }}' +
    '</div>' + 
  '</div>';
}

// later, in your columnDefs
{ field : 'balance', cellTemplate: getCellTemplate('balance1') }
```